server/pulp/server/async/app.py

```diff
@@ -49,6 +61,22 @@ def initialize_worker(sender, instance, **kwargs):
     """
     initialization.initialize()
 
+    if sender.startswith(constants.RESOURCE_MANAGER_WORKER_NAME):
+        msg = _("An instance of pulp_resource_manager is already running. "
+                "Halting execution of the dpulicate pulp_resource_monitor "
+                "until the original dies.")
+        _logger.info(msg)
```

Comment:
The message should probably be more along the lines of "An instance of pulp_resource_manager is already active. Halting execution until the currently active resource_manager becomes unavailable."